src/pf-accordion/index.html

```diff
@@ -0,0 +1,156 @@
+<!doctype html>
+<html>
+<head>
+  <title>pf-accordion example</title>
+  <link rel="stylesheet" href="/dist/css/patternfly.css">
```

Comment:
can we make all paths relative? i.e.`../../dist/css/patternfly.css`

This will enable sharing this page in your fork. More comments on that below...